app/controllers/events/volunteerRegisterEvent.py

```diff
@@ -0,0 +1,21 @@
+from app.models import*
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.user import User
+from app.controllers.events.meetsReqsForEvent import isEligibleForProgram
+import pytest
+
+def volunteerRegister(userid,  eventid):
```

Comment:
Same thing here, add some docstrings to explain what this function does.  It will help clarify what exactly is happening with the conditionals below.